.NET/Microsoft.Recognizers.Text.DateTime/DateTimeRecognizer.cs

```diff
@@ -95,6 +96,12 @@ protected override void InitializeConfiguration()
                     new BaseMergedDateTimeParser(
                         new GermanMergedParserConfiguration(new BaseOptionsConfiguration(options))),
                     new BaseMergedDateTimeExtractor(new GermanMergedExtractorConfiguration(options))));
-        }
+
+            RegisterModel<DateTimeModel>(

```

Comment:
We should only register the model to expose it to the outside once all tests for all specs pass (or the majority do and only corner cases or less common ones are left). Can you comment this one out?
Tests of all extractors/parsers are used to measure coverage. Once all look good, we add final model tests and enable it here.